src/diffusers/models/attention_processor.py

```diff
@@ -78,6 +78,10 @@ class Attention(nn.Module):
         only_cross_attention (`bool`, *optional*, defaults to `False`):
             Set to `True` to only use cross attention and not added_kv_proj_dim. Can only be set to `True` if
             `added_kv_proj_dim` is not `None`.
+        kv_heads (`int`,  *optional*, defaults to `None`):
+            The number of key and value heads to use for multi-head attention. Defaults to `heads`.
+            If `kv_heads=heads`, the model will use Multi Head Attention (MHA), if `kv_heads=1` the model will use 
+            Multi Query Attention (MQA) otherwise GQA is used.
```

Comment:
I've also added `kv_heads` for Grouped-Query Attention (GQA) support, it only works with `StableAudioAttnProcessor2_0` for now.

Let me know if that works for you!

Reply:
Oh right, `kv_heads`''ve just been added 3 days ago in #8652!

Contrarily to Lumina we don't use a ` s normalization layer` which is smthg I've never seen before!

(I can rename the stable audio one to `GroupedQueryAttnProcessor2_0` if necessary)